HIPs/incubation/identus.md

```diff
@@ -0,0 +1,172 @@
+
+Considering these insights, the Atala PRISM team developed a second project iteration. The new version focuses on compliance with W3C recommendations, supports some [Hyperledger Aries RFCs](https://github.com/hyperledger/aries-rfcs/blob/main/index.md), and adopted [DIDComm v2](https://identity.foundation/didcomm-messaging/spec/v2.0/). The project has evolved significantly and is on the path to support AnonCreds, SD+JWT, and OID4VC. What started as an API-driven Cloud Agent became extended by Wallet SDKs with a comprehensive support of languages, a standalone DIDComm v2 compatible Mediator, and a generic Cryptographic library providing a flexible foundation for decentralized identity solutions.
+
+As part of the project evolution, in 2023, IOG decided to open-source Atala PRISM's codebase and contribute its main component — the Cloud Agent (previously known as the Open Enterprise Agent) — to [Hyperledger Labs](https://wiki.hyperledger.org/display/labs). In the open-source space, we've engaged with the community, opened communication channels, and hosted regular community meetings, leading to improvements based on user feedback. Our efforts have encouraged the development of several third-party PoCs and nurtured a growing community. While the project still has a few references to its roots as an internal project from IOG, it is already mostly decoupled, and the development is happening not only by developers from IOG but also by the community.
+
+We will increase our involvement in Hyperledger by adding additional decentralized identity components to the Identus project, enabling us to reach a wider audience and promote technology adoption.
+
+## Relationship to Hyperledger Aries
+
+Identus builds upon the foundational protocols defined in Hyperledger Aries RFCs, which provide secure communication and verifiable credential exchange standards.  While adhering to these Aries standards, Identus offers a distinct implementation using the Scala programming language, which aligns with other Aries-inspired frameworks such as AcaPy (Python) and AFJ (JavaScript).
+
+A key difference lies in Identus' emphasis on DIDComm v2, the latest version of the DIDComm messaging protocol, while many existing Aries implementations currently focus on DIDComm v1.
+
+## Collaboration Opportunities
+
+Identus and Aries hold the potential for valuable collaboration. Identus' experience with DIDComm v2 could aid in its adoption within Aries. Additionally, exploring the integration of Aries Askar as a secure storage and key management solution for Identus could prove beneficial.
```

Comment:
Has any work been done on this collaboration while in Hyperledger Labs?

Reply:
No, these are simply items that have been identified as potential starting points for collaboration engagement